src/preprocessing.ml

```diff
@@ -203,7 +203,9 @@ let no_driver_error pps =
   | None ->
     sprintf
       "No ppx driver found.\n\
-       It seems that these ppx rewriters are not compatible with jbuilder."
+       It seems that these ppx rewriters are not compatible with Dune.\n\
```

Comment:
Rather than "these", should we just include the pps here in the error message (which also has locs)

Reply:
Indeed. We don't always have the location at this point because we might reach this program point when generating the rules for `_build/default/.ppx/<key>/ppx.exe`.

I'm refactoring a bit this code because it's becoming messy.